WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -93,7 +93,10 @@ final class CardPresentPaymentsOnboardingUseCase: CardPresentPaymentsOnboardingU
     }
 
     func selectPlugin(_ selectedPlugin: CardPresentPaymentsPlugin) {
-        assert(state == .selectPlugin)
+        assert(state == .selectPlugin(pluginSelectionWasCleared: true) ||
```

Comment:
I am not quite happy with this long assert, but I couldn't find any other way to assert all cases.

Reply:
I can't find any better direct way of doing it... but you could add this extension to the enum:

```
public var isSelectPlugin: Bool {
        if case .selectPlugin = self {
            return true
        } else {
            return false
        }
    }
```

... same as we have for `isCompleted`, and then just do 

`assert(state.isSelectPlugin)` which is at least easier to read?